app/assets/javascripts/tests/_draw_session.test.js

```diff
@@ -31,8 +31,35 @@ test('drawMobileSessionStartPoint calls drawMarker', t => {
   t.end();
 });
 
-const _drawSession = ({ map }) => {
-  const _map = { ...map };
-  const _heat = { getLevel: () => {} };
-  return drawSession(null, _map, _heat);
+test('drawFixedSession creates a marker if heat level is not null', t => {
+  const marker = {};
+  const heat = { getLevel: () => 1 }
+  const drawSessionMock = _drawSession({ heat });
+  const session = { last_hour_average: 1 };
+
+  const actual = drawSessionMock.drawFixedSession(session);
+
+  const expected = [marker]
+  t.deepEqual(actual, expected);
+
+  t.end();
+})
+
+test('drawFixedSession does not create a marker if heat level is null', t => {
+  const drawSessionMock = _drawSession({});
```

Comment:
Have you considered passing something like `const heat = { getLevel: () => null }`. Right now the test relies on the default which is hidden in `_drawSession`.

Reply:
Also, you may want to call it stub. Mock has another meaning. Maybe just service would do (we have been calling it service in other tests).

Stub is a test double that is instructed to return specific values.
Mock is a test double that performs expectations on calls.